src/lucky_migrator/create_table_statement.cr

```diff
@@ -148,11 +152,15 @@ class LuckyMigrator::CreateTableStatement
     end
   end
 
-  def references(table_name : String | Symbol | Nil)
-    if table_name
+  def references(table_name : String | Symbol | Nil, on_delete = :do_nothing)
+    if table_name.nil?
+      ""
+    elsif on_delete == :do_nothing
       " REFERENCES #{table_name}"
+    elsif ALLOWED_ON_DELETE_STRATEGIES.includes?(on_delete)
```

Comment:
You can get at the constant in another class like this `CreateForeignKeyStatement::ALLOWED_ON_DELETE_STRATEGIES`

Reply:
ah of course! Thanks